### PR TITLE
Add Login and SignUp page 

### PR DESCRIPTION
**Pull request by AnicetFantomas**
In this pull request we performed :

- Add login and signUp pages to the app
- Add mobile navigation bar 
- Add details for reservation page

### CODE REVIEW

File: src/components/AuthenticationNav/AutenticationNav.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        <img src={logo} className="logo rounded-full" alt="logo" />
+      </div>
+
+      <div className="navLinks flex flex-col  mt-44 items-center h-[10%] w-[100%] sm:pl-[15px] pb-12 pt-0">
+        <NavLink exact to="/" activeClassName="nav-active" className="rounded-lg border-transparent py-2 px-4 hover:bg-indigo-700 text-xl hover:text-[#fff] ease-in duration-300 sm:text-[90%] font-medium mb-2 ">Login</NavLink>
+        <NavLink exact to="/Signup" activeClassName="nav-active" className=" rounded-lg border-transparent py-2 px-4 hover:bg-indigo-700 text-xl hover:text-[#fff] ease-in duration-300 sm:text-[90%] font-medium mb-2 ">Sign up</NavLink>
+      </div>
+    </div>
+  );
+}
```
Comment (IfzaRasool) [inline src/components/AuthenticationNav/AutenticationNav.js]: This  AutenticationNav  serves no function here and it should be removed.

File: src/components/Authentication/LoginPage/LoginPage.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                <div className="flex items-center justify-between mt-4">
+                  <button type="submit" className="py-2 px-4  bg-indigo-600 hover:bg-indigo-700 focus:ring-indigo-500 focus:ring-offset-indigo-200 text-white w-full transition ease-in duration-200 text-center text-base font-semibold shadow-md focus:outline-none focus:ring-2 focus:ring-offset-2  rounded-lg ">
+                    Validate
+                  </button>
+                </div>
+                <div className="text-center">
+                  <Link to={SignUpPage} className="right-0 inline-block text-sm font-light align-baseline text-500 hover:text-gray-800">
+                    Create an account
+                  </Link>
+                </div>
```
Comment (IfzaRasool) [inline src/components/Authentication/LoginPage/LoginPage.js]: there should be correct link address.

File: src/components/Authentication/SignUpPage/SignUpPage.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                <button type="submit" className="py-2 px-4  bg-indigo-600 hover:bg-indigo-700 focus:ring-indigo-500 focus:ring-offset-indigo-200 text-white w-full transition ease-in duration-200 text-center text-base font-semibold shadow-md focus:outline-none focus:ring-2 focus:ring-offset-2  rounded-lg ">
+                  Sign Up
+                </button>
+              </div>
+              <div className="text-center">
+                <a href="vid.com" className="right-0 inline-block text-sm font-light align-baseline text-500 hover:text-gray-800">
+                  Login
+                </a>
+              </div>
+            </form>
```
Comment (IfzaRasool) [inline src/components/Authentication/SignUpPage/SignUpPage.js]: Make Sure the link address should be correct.

**Review by IfzaRasool — CHANGES_REQUESTED** [review]
Hello @AnicetFantomas . You've done Good job so far.

Here are some observations I could get from your code asides from the ones I mentioned as inline comments.
Kindly review them, thanks. Let me know if you have any questions

File: src/components/AuthenticationNav/AutenticationNav.js (hunk 1 above)
Comment (deyemiobaa) [reply src/components/AuthenticationNav/AutenticationNav.js]: @IfzaRasool This authentication nav is quite important. This is what we want to render when the isn't logged in

Comment (deyemiobaa) [reply src/components/AuthenticationNav/AutenticationNav.js]: I'll go ahead and pull this branch anyway. I'll fix the other issues you mentioned and merge the branch

**Review by deyemiobaa — APPROVED** [review]
Hello Team. I'll close this PR now. I've fixed the UI and made a couple other changes like reusing components to make the code better.

**Pull request merged**